app/models/concerns/smooch_zendesk.rb

```diff
@@ -69,12 +69,16 @@ def zendesk_send_message_to_user(uid, text, extra = {}, _force = false)
           }
         })
       end
-      return if params['type'] == 'text' && params['text'].blank?
+      return OpenStruct.new(body: 'Empty message', code: 400) if params['type'] == 'text' && params['text'].blank?
```

Comment:
Return an object that looks like a HTTP response (e.g., contains code and body).